MoltenVKShaderConverter/MoltenVKShaderConverter/GLSLToSPIRVConverter.cpp

```diff
@@ -21,7 +21,7 @@
 #include "SPIRVToMSLConverter.h"
 #include "SPIRVSupport.h"
 #include "MVKStrings.h"
-#include <glslang/SPIRV/GlslangToSpv.h>
+#include <SPIRV/GlslangToSpv.h>
```

Comment:
```suggestion
#include <glslang/SPIRV/GlslangToSpv.h>
```

Reply:
I'm not sure I properly understand the distinction between the two environments now.

For both `glslang` and `SPIRV-Cross`, pulling the Git repo, creating a `mybuild` directory inside it (or somewhere else), and running `cmake` and `make` into it, leaves all the headers relative to the base directory of the respective Git repos. 

So to be consistent, we now point MoltenVK header paths to those base directories: `glslang` and `SPIRV-Cross`, respectively. And MoltenVK `include` statements are now relative to those base directories.

Help me understand what I'm missing about your integrated build environment. How does this work there? Why does the header path for `SPIRV-Cross` point to its Git repo directory, but the header path for `glslang` point to the parent directory of its Git repo directory?

Reply:
Build folder layout and official include directories of a lib are orthogonal.

I rely on installation folders and official CMake config files installed with those libs which (should) contain the truth about how to consume these libraries, not build folder which is just an intermediate stuff.

The layout of build subfolders might be completly different from installation layout. Package managers rely on installation layout, and actually they rely on pkg-config/CMake config files information if any (or just a README explaining how to include headers, or just a consensus in the ecosystem).

In my build environment I use a package manager (conan) for my dependencies.
I've created a package for MoltenVK in public repository of conan, which has removed all the vendoring mechanism of MoltenVK because it's a "no go" (or something to remove quickly to avoid side effects like ODR violation, non reproducible builds etc) for a package manager.
Therefore in this context, dependencies of MoltenVK are installed by this package manager, which honors official install layout of each lib. This is why I had to patch includes of spirv-cross in MoltenVK.

The official way to include:
- `glslang` public headers is not `<SPIRV/GlslangToSpv.h>` but `<glslang/SPIRV/GlslangToSpv.h>` (the truth is in CMake config file installed with glslang).
- `spirv-cross` public headers is not `#include <SPIRV-Cross/spirv.hpp>` (this folder doesn't exist actually when you build and install yourself spirv-cross, it's `spirv_cross` AFAIK), not even `#include <spirv_cross/spirv.hpp>` but `#include <spirv.hpp>`.
  So here for example you see that even if they have added a "namespaced folder", authors of spirv-cross have decided that this folder will be useless in a consuming point of view.

Even install layout might be misleading, the truth (include direcrtories, compile flags, dependencies etc) of CMake based projects is in CMake config or pkg-config file installed along headers and binaries.

Reply:
Done.